content/docs/command-reference/add.md

```diff
@@ -148,6 +164,18 @@ not.
   > Note that external outputs typically require an external cache setup. See
   > link above for more details.
 
+- `--to-remote` - Import an external target, but don't move it into the
```

Comment:
💅 Import -> import (and in other places I think)

Reply:
```suggestion
- `--to-remote` - import an external target, but don't move it into the
```